packages/sku/package.json

```diff
@@ -136,14 +137,14 @@
     "webpack-dev-server": "^5.0.2",
     "webpack-merge": "^5.8.0",
     "webpack-node-externals": "^3.0.0",
-    "wrap-ansi": "^7.0.0",
-    "default-browser": "^5.2.1"
+    "wrap-ansi": "^7.0.0"
   },
   "devDependencies": {
     "@types/cross-spawn": "^6.0.3",
     "@types/debug": "^4.1.12",
     "@types/eslint": "^8.56.10",
     "@types/minimist": "^1.2.5",
+    "@types/node": "^18.19.31",
```

Comment:
This is already a monorepo root dependency, but I figured it's worth being thorough about which types the `sku` CLI uses.